hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java

```diff
@@ -12,7 +12,8 @@
 
     ZOOKEEPER_CONNECT_STRING("zookeeper.connect.string", "localhost:2181"),
     ZOOKEEPER_BASE_SLEEP_TIME("zookeeper.base.sleep.time", 1000),
-    ZOOKEEPER_MAX_SLEEP_TIME("zookeeper.max.sleep.time.seconds", 30),
+    ZOOKEEPER_MAX_SLEEP_TIME_IN_SECONDS("zookeeper.max.sleep.time.seconds", 30),
+    ZOOKEEPER_MAX_RETRIES("zookeeper.max.retries", 29),
```

Comment:
Decreasing to 29 as this is maximum value which can be used in ExponentialBackoffRetry.